conf/config.yaml

```diff
@@ -55,6 +55,13 @@ apisix:
   #   - "::/64"
   # port_admin: 9180              # use a separate port
 
+  # allow get nginx status data
+  allow_status:                 # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
+    - 127.0.0.0/24              # If we don't set any IP list, then any IP access is not allowed by default.
```

Comment:
According to your code implementation, if you do not set any ip, the default is allow 127.0.0.1/24

Reply:
what do you mean? shouldn't i commit this config file?

Reply:
Make sure that the comment is the same as the actual effect. @ywengineer any news?